modules/apps/frontend-view-state/frontend-view-state-service/src/main/resources/META-INF/portlet-model-hints.xml

```diff
@@ -53,10 +53,16 @@
 		<field name="userName" type="String" />
 		<field name="createDate" type="Date" />
 		<field name="modifiedDate" type="Date" />
-		<field name="fdsName" type="String" />
+		<field name="fdsName" type="String">
```

Comment:
we have some predefined models that can be used that sets the max-length and other properties in case it is interesting for you to use
https://github.com/liferay/liferay-portal/blob/639ab1904eb6fe3f3534291074daf7bf26f220ed/portal-impl/src/META-INF/portal-model-hints.xml

Reply:
The most common form an `fdsName` in DXP has is portlet key + some short string, unique to portlet. See for example [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/constants/FDSSampleFDSNames.java#L28-L29). In the linked example, `FDSSampleFDSNames.CUSTOMIZED` will resolve to
```
com_liferay_frontend_data_set_sample_web_internal_portlet_FDSSamplePortlet-customized
```

We are setting FDS names this way to ensure they are unique in DXP. We need uniqueness for registry mechanisms of views, filters, etc.

So, if it standard for `portletId` in models to be 200, my thinking is `fdsName` needs to be 200 or more. We can set to some arbitrarily bigger number, like 250, but 200 is probably fine.

Reply:
Understood. Just mentioning in case we can refer to some already defined constant or type of model to avoid just giving an arbitrary size.